src/flamegraph/color/palettes.rs

```diff
@@ -59,6 +59,31 @@ pub(super) mod perl {
     }
 }
 
+pub(super) mod python {
+    use crate::flamegraph::color::BasicPalette;
+
+    fn split_any_path(path: &str) -> impl Iterator<Item = &str> {
+        path.split(|c| c == '/' || c == '\\')
+    }
+
+    pub(in super::super) fn resolve(name: &str) -> BasicPalette {
+      if split_any_path(name).any(|part| part == "site-packages") {
+            return BasicPalette::Mem;
+        } else if split_any_path(name).any(|part| {
+            part.strip_prefix("python")
+                .or_else(|| part.strip_prefix("Python"))
+                .map_or(false, |version| {
+                    version.chars().all(|c| c.is_digit(10) || c == '.')
+                })
+        }) || name.starts_with("<")
```

Comment:
Why is this clause here? I see the test string below, but are those _always_ standard library? Is there something more we can test on than _just_ `<`?

Reply:
The angle bracket syntax is used internally by cpython, generally for built-ins, methods on built-in objects, import machinery, etc. For normal user code the name should be the path which should never hit this case. See e.g. here:
https://github.com/python/cpython/blob/main/Python/import.c#L1469

Reply:
Well, so, for example, is it always `<frozen ...`? If so, could we match on that?

Reply:
I'll do a bit of digging - the two I see often are "<frozen ..." and "<built-in ..." so potentially we could just match on those two

Reply:
Narrowed it down to `<frozen`, `<built-in` and `<method`. I've updated the PR to target these specifically